docs/_sass/_style.scss

```diff
@@ -1031,3 +1031,11 @@ code.output {
   clip: rect(0, 0, 0, 0);
   border: 0;
 }
+
+.result {
+  border: 1px solid yellow;
```

Comment:
Thanks for the comments. I'll make the updates in a couple of days (I'm finishing up some presentations).